#1 Homework

```diff
@@ -0,0 +1,27 @@
+var array = [3, 5, 8, 2, 1, 0, 0, 4]
+
+var a = 0;
+var b = array[0];
+var c = 0;
+var maxd = 0;
+var d = array[0];
+
+for (var i = 0; i < array.length; i++) {
+    a = array[i] + a; // a = hodnota v poli + a. Po prvni iteraci; a = 3 + 0 -> a = 3. Po druhe iteraci; a = 5 + 3 -> a = 8.
```

Comment:
Please use comments in english, it will help you to memorise correct terms faster

Reply:
- all comments translated